content/_releases/23/31.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Product Update - v23.31
+description: Details of the product update v23.31 from 1st of August 2023.
+layout: rel
+version: "23.31"
+section: Product Updates
+category: product-updates
+releaseDate: 2023-08-01
+redirect_from:
+  - /releases/23.31.html
+---
+
+## New Features
+### Dashboard Redesign:
+* Added Ordinary Flows counter - *click on the counter to open `Flows` page with filtered ordinary flows*
+* Added Real-time Flows counter - *click on the counter to open `Flows` page with filtered real-time flows*
+* Redesigned the current Execution Statistics graph - *each day on the graph is clickable and leads users to the corresponding page*
+* Added new graph for Container errors - *each day on the graph is clickable and leads users to the corresponding page*
+* Transferred Workspace Memory Usage graph
+
+### Transfer modal windows to the new styles:
+* Transform UI for Flow Designer: Configuration for Content-Based Router Component to React - Now users can map values from previous steps for evaluation.
```

Comment:
Flow designer UI changes: Configuration screens for the Content Based Router are changed to Reat & now Users can map values from previous steps for evaluation